File: Basic/Test_FizzBuzz.c
#include <stdio.h>

void Solution(int n)
{
    for (int i = 1; i < n + 1; i++)
    {
        if (i % 3 == 0 && i % 5 == 0)
        {
            printf("FizzFuzz ");
        }
        else if (i % 3 == 0)
        {
            printf("Fizz ");
        }
        else if (i % 5 == 0)
        {
            printf("Buzz ");
        }
        else
        {
            printf("%d ", i);
        }
    }
}

int main()
{
    int t1 = 15;
    Solution(t1);
    return 0;
}

File: Basic/Test_FizzBuzz.py
def Solution(n=int):
    li = []
    for i in range(1, n + 1):
        if i % 3 == 0 and i % 5 == 0:
            li.append('FizzBuzz')
        elif i % 3 == 0:
            li.append('Fizz')
        elif i % 5 == 0:
            li.append('Buzz')
        else:
            li.append(str(i))
    return li


print(Solution(n=15))
